src/gen/models/index.ts

```diff
@@ -7196,6 +7196,8 @@ export interface MessageDeletedEvent {
 
   thread_participants?: User[];
 
+  deleted_for_me?: boolean;
+
   message?: Message;
```

Comment:
Was this change generated with `yarn generate:open-api`? Shouldn't we have a flag in `DeleteMessageRequest`?